index.js

```diff
@@ -6,7 +6,7 @@
 var client = require('redis').createClient;
 var parser = require('socket.io-parser');
 var hasBin = require('has-binary');
-var msgpack = require('msgpack-lite');
+var notepack = require('notepack.io');
```

Comment:
It looks like a wise choice! like socket.io-redis (https://github.com/socketio/socket.io-redis/blob/master/index.js#L8)